combat-trainer.lic

```diff
@@ -1844,6 +1874,7 @@ class GameState
     @aim_queue = []
     @dance_queue = []
     @analyze_combo_array = []
+    @currently_whirlwinding = false
```

Comment:
Can you add these new vars to the declarations? In my copy that's lines 1812, 2326, or 2327.